### ISSUE TITLE
"django.core.exceptions.AppRegistryNotReady: Models aren't loaded yet" when upgrading from 1.6.0 to 1.7.X

### ISSUE THREAD

**Issue by lucacorti** [GetBlimp/django-rest-framework-jwt#153]
I'm on Django 1.8.4, DRF 3.2.2, DRF-JWT 1.6.0.

When trying to upgrade to 1.7.0 or 1.7.1 I get:

```
 Traceback (most recent call last):
   File "manage.py", line 10, in <module>
     execute_from_command_line(sys.argv)
   File "lib/python2.7/site-packages/django/core/management/__init__.py", line 338, in execute_from_command_line
     utility.execute()
   File "lib/python2.7/site-packages/django/core/management/__init__.py", line 312, in execute
     django.setup()
   File "lib/python2.7/site-packages/django/__init__.py", line 18, in setup
     apps.populate(settings.INSTALLED_APPS)
   File "lib/python2.7/site-packages/django/apps/registry.py", line 108, in populate
     app_config.import_models(all_models)
   File "lib/python2.7/site-packages/django/apps/config.py", line 198, in import_models
     self.models_module = import_module(models_module_name)
   File "/usr/local/lib/python2.7/importlib/__init__.py", line 37, in import_module
     __import__(name)
   File "/usr/home/snap/snap/conversations/models.py", line 20, in <module>
     from snap.api.exceptions import SnapAPIError
   File "/usr/home/snap/snap/api/exceptions.py", line 8, in <module>
     from rest_framework.views import exception_handler
   File "lib/python2.7/site-packages/rest_framework/views.py", line 98, in <module>
     class APIView(View):
   File "lib/python2.7/site-packages/rest_framework/views.py", line 103, in APIView
     authentication_classes = api_settings.DEFAULT_AUTHENTICATION_CLASSES
   File "lib/python2.7/site-packages/rest_framework/settings.py", line 208, in __getattr__
     val = perform_import(val, attr)
   File "lib/python2.7/site-packages/rest_framework/settings.py", line 160, in perform_import
     return [import_from_string(item, setting_name) for item in val]
   File "lib/python2.7/site-packages/rest_framework/settings.py", line 172, in import_from_string
     module = importlib.import_module(module_path)
   File "/usr/local/lib/python2.7/importlib/__init__.py", line 37, in import_module
     __import__(name)
   File "lib/python2.7/site-packages/rest_framework_jwt/authentication.py", line 13, in <module>
     User = get_user_model()
   File "lib/python2.7/site-packages/rest_framework_jwt/compat.py", line 29, in get_user_model
     User = get_user_model()
   File "lib/python2.7/site-packages/django/contrib/auth/__init__.py", line 150, in get_user_model
     return django_apps.get_model(settings.AUTH_USER_MODEL)
   File "lib/python2.7/site-packages/django/apps/registry.py", line 199, in get_model
     self.check_models_ready()
   File "lib/python2.7/site-packages/django/apps/registry.py", line 131, in check_models_ready
     raise AppRegistryNotReady("Models aren't loaded yet.")
 django.core.exceptions.AppRegistryNotReady: Models aren't loaded yet.
```

**Comment by lucacorti** [comment]
I think this is #152

**Comment by jpadilla** [comment]
Closed via #152
